client-cm/src/main/java/com/sequenceiq/cloudbreak/cm/client/ClouderaManagerApiClientProvider.java

```diff
@@ -33,6 +34,8 @@ public class ClouderaManagerApiClientProvider {
 
     public static final String API_V_51 = API_ROOT + "/v51";
 
+    public static final String API_V_52 = API_ROOT + "/v52";
+
```

Comment:
What is this needed for? The logic in `ClouderaManagerSetupService.initApiClient()` seems to depend on v51 for the banner call.